addons/account/static/tests/reconciliation_tests.js

```diff
@@ -225,9 +232,16 @@ var db = {
             if (!Datas.used.move_lines_for_manual_reconciliation[key]) {
                 throw new Error("Unknown parameters for get_move_lines_for_manual_reconciliation: '"+ key + "'");
             }
-            return $.when(_.filter(Datas.used.move_lines_for_manual_reconciliation[key], function (line) {
-                return excluded_ids.indexOf(line.id) === -1;
-            }));
+            var lines = Datas.used.move_lines_for_manual_reconciliation[key]
+                .filter(function (line) {
+                    return excluded_ids.indexOf(line.id) === -1;
+                })
+                .map(function (line, i, src) {
+                    line.recs_count = src.length;
+                    return line;
+                })
+                .slice(0, options.params.limitMoveLines);
```

Comment:
are you sure options.params.limitMoveLines is defined? where is options defined?